optuna_dashboard/llm/_prompt_templates.py

```diff
@@ -0,0 +1,86 @@
+TRIAL_FILTERING_PROMPT_TEMPLATE = """Please write a filtering function for the user query provided in Javascript.
```

Comment:
To prevent generating unexpected or unsafe JavaScript functions, could you update the prompt template to explicitly prohibit all types of network requests (e.g., fetch, XMLHttpRequest, form submissions via DOM manipulation, etc.) as well as any behavior that could potentially harm or mislead the user?